packages/core/template/project/DevOps.md

```diff
@@ -0,0 +1,86 @@
+---
+  layout: default.md
+  title: "DevOps guide"
+  pageNav: 3
+---
+
+# DevOps guide
+
+<!-- * Table of Contents -->
+<page-nav-print />
+
+<!-- -------------------------------------------------------------------------------------------------------------------- -->
+
+<box type="tip">
+This page can be used to understand the DevOps practices adopted by the project.
+</box>
+
+## Build automation
+
+This example project uses Gradle for **build automation and dependency management**. **You are recommended to read [this Gradle Tutorial from the se-edu/guides](https://se-education.org/guides/tutorials/gradle.html)**.
+
+
+Given below are how to use Gradle for some important project tasks.
+
+
+* **`clean`**: Deletes the files created during the previous build tasks (e.g. files in the `build` folder).<br>
```

Comment:
I would delete a lot of this example content - You can leave in the headers, and maybe a bit to demonstrate functionality of MarkBind (like code formatting for example or internal link like how you did here with Gradle) but remove some of the specific points about code cov / gradle.